tests/acceptance/10_files/rename/main.cf

```diff
@@ -0,0 +1,92 @@
+body file control
+{
+        inputs => { "../../default.cf.sub" };
+}
+
+bundle agent main
+{
+  methods:
+      "init,test,check,cleanup"
+        usebundle => default("$(this.promise_filename)") ;
+}
+
+bundle agent init
+{
+  files:
+
+      # First we initialize some directories to iterate over and some files that
+      # will be renamed during the test
+
+      "$(G.testdir)/test/$(check.dirs)/incoming/$(check.files)"
+        create => "true";
+}
+
+bundle agent test
+{
+  meta:
+
+      "description"
+        string => "Test that files can be renamed using regular expression
+        matches from the promiser";
+
+      "test_soft_fail"
+        string => "any",
+        meta => { "CFE-2879" };
+
+  files:
+
+      # We promise that the target is a copy of the source file Since preserve
+      # is set to false, we expect the original permissions on the target file
+      # to remain unchanged.
```

Comment:
Isn't this a copy-paste from a different test?

Reply:
No, I don't think it is from a different test.